### ISSUE TITLE
404 Not Found: architect-http-static

### ISSUE THREAD

**Issue by shubhendu-abhishek** [c9/architect#69]
npm ERR! code E404
npm ERR! 404 Not Found: architect-http-static@~0.0.1

While doing npm  install getting above error

**Comment by npatel33** [comment]
Yes, I am facing the same issue. Did you figure it out? I am also having problem with "architect-http". While loading config it says "Module not found!"

**Comment by arucil** [comment]
`Architect-http-static` and `architect-http` modules resides in `node_modules` already, before running `npm install` you should delete those two dependencies from `package.json` and copy the two module folders somewhere to prevent npm from deleting them. After npm install is done, copy back the two modules.

**Comment by greuze** [comment]
I had the same issue here. You can do as @arucil explains, or "convert" those two modules in local plugins, by moving to `plugins` folder, removing dependencies from root `package.json` and updating the path in `config.js` (i.e.: `architect-http-static` -> `./plugins/architect-http-static`)
